Makefile

```diff
@@ -104,6 +126,21 @@ SASS_SPEC_SPEC_DIR ?= spec
 SASSC_BIN = $(SASS_SASSC_PATH)/bin/sassc
 RUBY_BIN = ruby
 
+LIB_STATIC = $(SASS_LIBSASS_PATH)/lib/libsass.a
+LIB_SHARED = $(SASS_LIBSASS_PATH)/lib/libsass.so
+
+ifeq (MinGW,$(UNAME))
+	ifeq (shared,$(BUILD))
+		CFLAGS     += -D ADD_EXPORTS
+		CXXFLAGS   += -D ADD_EXPORTS
+		LIB_SHARED  = $(SASS_LIBSASS_PATH)/lib/libsass.dll
+	endif
+else
+	CFLAGS   += -fPIC
+	CXXFLAGS += -fPIC
```

Comment:
Yeah it sends to whatever compiler is being used. So no matter the language, the same params are sent. SO: http://stackoverflow.com/a/495646/217965 

The language in the wikipedia page is super confusing. The CGO docs are way more clear (handles C/C++ compiling/linking for Go)

> All the cgo CPPFLAGS and CFLAGS directives in a package are concatenated and used to compile C files in that package. All the CPPFLAGS and CXXFLAGS directives in a package are concatenated and used to compile C++ files in that package.